src/hotspot/share/utilities/vmError.cpp

```diff
@@ -573,7 +573,7 @@ void VMError::report(outputStream* st, bool _verbose) {
     if (_verbose && TestCrashInErrorHandler == TEST_RESOURCE_MARK_CRASH) {
       stringStream message;
       message.print("This is a message with no ResourceMark");
-      tty->print_cr("%s", message.as_string());
+      tty->print_cr("%s", message.base());
```

Comment:
I think the intent was to use as_string deliberately to get an mark-missing crash.

Reply:
Thank you!